tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -323,27 +324,29 @@ public class LinkModeFullClass {
 								return sr.ReadToEnd ();
 						},
 					},
-					new BuildItem.Source ("Bug35195.cs") {
-						TextContent = () => {
-							using (var sr = new StreamReader (typeof (InstallAndRunTests).Assembly.GetManifestResourceStream ("Xamarin.Android.Build.Tests.Resources.LinkDescTest.Bug35195.cs")))
-								return sr.ReadToEnd ();
-						},
-					},
 					new BuildItem.Source ("Bug36250.cs") {
 						TextContent = () => {
 							using (var sr = new StreamReader (typeof (InstallAndRunTests).Assembly.GetManifestResourceStream ("Xamarin.Android.Build.Tests.Resources.LinkDescTest.Bug36250.cs")))
 								return sr.ReadToEnd ();
 						},
 					},
 				},
-				PackageReferences = {
-					new Package {
-						Id = "sqlite-net-pcl",
-						Version = "1.7.335",
-					},
-				},
 			};
 
+			// sqlite-net-pcl does not work in .NET 6
+			if (!Builder.UseDotNet) {
+				lib2.PackageReferences.Add (new Package {
+					Id = "sqlite-net-pcl",
+					Version = "1.7.335",
+				});
+				lib2.Sources.Add (new BuildItem.Source ("Bug35195.cs") {
+					TextContent = () => {
+						using (var sr = new StreamReader (typeof (InstallAndRunTests).Assembly.GetManifestResourceStream ("Xamarin.Android.Build.Tests.Resources.LinkDescTest.Bug35195.cs")))
+							return sr.ReadToEnd ();
+					},
+				});
+			}
+
```

Comment:
There seem to be a few issues when `AndroidLinkMode=Full` in the CustomLinkDescriptionPreserveFull test: https://github.com/xamarin/xamarin-android/blob/master/tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs#L275

```
11-11 23:06:06.463  8520  8520 I XALINKERTESTS: [FAIL] Unable to create instance of 'SomeClass'.
```

We might want to ignore this test parameter for One .NET for now? I think there was some discussion about simple reflection based linker preservation not working? Maybe @radekdoulik can speak more to this.

Reply:
I think this is because we don't pass the custom linker xml's to ILLink. I have opened https://github.com/xamarin/xamarin-android/issues/5288 for that. Once this is fixed, the test should hopefully pass.

Reply:
I thought this passed for me locally... I'll put `[Category("DotNetIgnore"]` for now and link to #5288.

Reply:
It looks like it did pass for the AndroidLinkMode=SdkOnly, but not AndroidLinkMode=Full. Maybe we should add an Assert.Ignore when the linker configuration parameter is `Full` && `Builder.UseDotNet`?